src/fhir/testUtilities.ts

```diff
@@ -0,0 +1,55 @@
+import { Bundle } from "@smile-cdr/fhirts/dist/FHIR-R4/classes/bundle";
+import { checkResponseOK } from "./api";
+import { FHIR_URL } from "./api.test";
```

Comment:
having to import something from a test is probably an antipattern, mentioned before that I think the FHIR url should be defined in this file